pkg/auth/auth_type.go

```diff
@@ -0,0 +1,30 @@
+package auth
+
+import (
+	"context"
+	"google.golang.org/grpc/metadata"
+)
+
+type AUTH_TYPE int
```

Comment:
AuthType ?

Reply:
yes, it's possible to have token auth or session based